unison-src/transcripts/edit-namespace.output.md

```diff
@@ -12,43 +12,76 @@ simple.y = 20
 
 -- Shouldn't edit things in lib
 lib.project.ignoreMe = 30
-```
 
-Edit current namespace
+-- Shouldn't render record accessors
+unique type Foo = { bar : Nat, baz : Nat }
+```
 
 ```ucm
-.simple> edit.namespace
 
-  ☝️
-  
-  I added 2 definitions to the top of scratch.u
+  Loading changes detected in scratch.u.
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
   
-  You can edit them there, then run `update` to replace the
-  definitions currently in this namespace.
+    ⍟ These new definitions are ok to `add`:
+    
+      type Foo
+      Foo.bar               : Foo -> Nat
+      Foo.bar.modify        : (Nat ->{g} Nat) -> Foo ->{g} Foo
+      Foo.bar.set           : Nat -> Foo -> Foo
+      Foo.baz               : Foo -> Nat
+      Foo.baz.modify        : (Nat ->{g} Nat) -> Foo ->{g} Foo
+      Foo.baz.set           : Nat -> Foo -> Foo
```

Comment:
It'll be great once we have first-class fields and these aren't listed here either!